source/mainnet/net/resources/release-notes.rst

```diff
@@ -11,6 +11,19 @@ Release Notes - Testnet
 
    To subscribe to updates on the Testnet status page click **Subscribe** to get all updates or click **Get updates** to choose to get all updates or only updates for specific products.
 
+March 10, 2023
+
+Concordium Node 5.2.X
+---------------------
+
+- The Prometheus metrics exporter has been cleaned up and improved in this release. Some metrics that were never updated were removed, a number of metrics were renamed, and behavior of some metrics was changed. See docs/prometheus-exporter.md for more details.
```

Comment:
```suggestion
- The Prometheus metrics exporter have been improved and systematized. The metrics are now `documented <https://github.com/Concordium/concordium-node/blob/main/docs/prometheus-exporter.md>`_ and the node's Prometheus metrics API stability will adhere to SEMVER guidelines.
```

(not sure if i did the link correctly, please check)

Reply:
What I wrote is a bit clumsy. But I wanted to convey that this is a stable API that people can use for monitoring from this release onwards.

Can you see a way to improve it @dg-concordium ?

Reply:
@abizjak : So "has" is actually correct because it has to be in agreement with exporter which is singular. Here is what I added: The Prometheus metrics exporter has been improved and systematized, making this API stable from this release onwards to monitor your node metrics. The metrics are now `documented <https://github.com/Concordium/concordium-node/blob/main/docs/prometheus-exporter.md>`_ and the node's Prometheus metrics API stability will adhere to SEMVER guidelines.